pkg/apis/v1alpha2/crd_validation_cel_test.go

```diff
@@ -122,6 +127,327 @@ var _ = Describe("CEL/Validation", func() {
 		})
 	})
 
+	Context("LocalDNS", func() {
+		DescribeTable("should validate LocalDNSMode", func(mode *v1alpha2.LocalDNSMode, expected bool) {
```

Comment:
minor: expected... what?
```suggestion
		DescribeTable("should validate LocalDNSMode", func(mode *v1alpha2.LocalDNSMode, expectedSuccess bool) {
```

Reply:
i went with your other suggestion on the same test: https://github.com/Azure/karpenter-provider-azure/pull/1233#discussion_r2586735539